pytorch_blade/benchmark/TorchBench/test_torch_bench.sh

```diff
@@ -34,20 +34,28 @@ ln -s $benchmark_repo_dir torchbenchmark
 # setup benchmark env
 export DISC_EXPERIMENTAL_SPECULATION_TLP_ENHANCE=true \
     DISC_CPU_LARGE_CONCAT_NUM_OPERANDS=4 DISC_CPU_ENABLE_EAGER_TRANSPOSE_FUSION=1 \
-    OMP_NUM_THREADS=1 TORCHBENCH_ATOL=1e-2 TORCHBENCH_RTOL=1e-2
+    TORCHBENCH_ATOL=1e-2 TORCHBENCH_RTOL=1e-2
 
 config_file=blade_$1_$2.yaml
 bench_target=$2
+binding_cores=1
 
 if [ $1 == "cpu" ]
 then
-    # 4 cores
-    export GOMP_CPU_AFFINITY="2-5"
+    if [ ! -n "$3" ]
+    then
+        export OMP_NUM_THREADS=1 GOMP_CPU_AFFINITY=4
```

Comment:
Why using different default value for `GOMP_CPU_AFFINITY ` and `taskset`?

Reply:
the origin code using in this way.  Now I using the same value